scripts/seed_demo_data.py

```
@@ -15,11 +18,24 @@
+# Load .env from project root to support Codespaces/devcontainer auto-creation
+project_root_env = Path(__file__).resolve().parents[1] / '.env'
+if project_root_env.exists():
+  load_dotenv(dotenv_path=project_root_env)
+
+# Fallback to environment variables if CLI params not provided
+if not args.posthog_api_key:
+  args.posthog_api_key = os.getenv("PH_PROJECT_KEY")
+if not args.posthog_host:
+  args.posthog_host = os.getenv("PH_HOST")
+if not args.posthog_api_key or not args.posthog_host:
+  raise SystemExit("PH_PROJECT_KEY and PH_HOST must be provided via flags or environment variables.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)